packages/block-editor/src/components/alignment-toolbar/index.js

```diff
@@ -46,11 +46,17 @@ export function AlignmentToolbar( props ) {
 		( control ) => control.align === value
 	);
 
+	const POPOVER_PROPS = {
```

Comment:
We should move this outside the component because it's a static object that shouldn't be created on each render.